### DISCUSSION TITLE
Welcome to splunk-otel-java Discussions!

### DISCUSSION THREAD

**Discussion by iNikem** [signalfx/splunk-otel-java#113] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by furkanali0325** [comment]
Hi @iNikem ,
I have integrated Splunk OTEL jar( open telemetry library) in my spring boot application for tracing and matrices.
So there is a SPLUNK_ACCESS_TOKEN that needs to be exposed to send data directly to the Splunk observability cloud. Currently, I have stored it in Kubernetes.yaml file. So I just want to know other approaches to expose this Splunk access token.

For example, there is a flag variable "OTEL_JAVAAGENT_ENABLED" used by the Splunk OTEL library and I have stored this value to "true" in Kubernetes.yaml file. So by this config OTEL jar is able to access it.

Is there any alternate way to store the splunk_access_token apart from Kubernetes.yaml or properties file?
